bundles/org.openhab.binding.revogi/src/main/java/org/openhab/binding/revogi/internal/udp/UdpResponse.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.revogi.internal.udp;
+
+import java.util.Objects;
+
+/**
+ * The class {@link UdpResponse} represents udp reponse we expect
+ *
+ * @author Andi Bräu - Initial contribution
+ */
+public class UdpResponse {
```

Comment:
You could append `DTO` to the class name to avoid the warnings about missing NonNullByDefault annotation.

Reply:
thanks for that hint, I didn't know that